charts/github-bot-safe-settings/templates/app-env.yaml

```diff
@@ -0,0 +1,22 @@
+{{- $fullName := include "safe-settings.fullname" . -}}
+{{- if .Values.appEnv.create }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "safe-settings.fullname" . }}-app-env
+type: Opaque
+data:
+  APP_ID: {{ required "A valid appEnv.APP_ID is required!" .Values.appEnv.APP_ID | toString | b64enc }}
+  PRIVATE_KEY: {{ required "A valid appEnv.PRIVATE_KEY is required!" .Values.appEnv.PRIVATE_KEY | b64enc }}
+  WEBHOOK_SECRET: {{ required "A valid appEnv.WEBHOOK_SECRET is required!" .Values.appEnv.WEBHOOK_SECRET | b64enc }}
+  {{- if .Values.appEnv.DEPLOYMENT_CONFIG_FILE }}
+  DEPLOYMENT_CONFIG_FILE: {{ .Values.appEnv.DEPLOYMENT_CONFIG_FILE | b64enc }}
+  {{- end }}
+  ADMIN_REPO: {{ .Values.appEnv.ADMIN_REPO | default "admin" | b64enc }}
```

Comment:
Let's add defaults in values.yaml itself instead of configuring it in templates, because then it would be much easier to see them from `values.yaml` (i.e. from one place) and not search for the place where it is used, i.e. secret or deployment or smth else

so please add a separate block in values.yaml like this:

```
appEnvSecret:
  create: false
  data:
    APP_ID: ""
    PRIVATE_KEY: ""
    WEBHOOK_SECRET: ""
    DEPLOYMENT_CONFIG_FILE: ""
    ADMIN_REPO: "admin"
    SETTINGS_FILE_PATH: "settings.yaml"
    ...
```

Reply:
resolve cause it is fork